### DISCUSSION TITLE
Remote upload limit

### DISCUSSION THREAD

**Discussion by hawk0484** [wiremod/wire#2638] [Suggestions]
It seems you can only remoteSetCode once per iteration, is it possible to have this limit as a server var?

**Comment by adosikas** [comment]
Limit seems to be about 1 / second and player, https://github.com/wiremod/wire/blob/master/lua/entities/gmod_wire_expression2/core/custom/remoteupload.lua#L5

IMO the limit makes sense for remoteUpload as it triggers the client to send files, but i don't think such a low limit is required for remoteSetCode, as that is serverside only.

**Comment by Divran** [comment]
setCode still needs a limit, since compiling etc is a lot of work. Maybe you're right and 1 second is a bit too much, but I'm not convinced. Sending the code from the client to the server is probably less work than compiling it.

**Comment by thegrb93** [comment]
This should be using e2's built in antispam shouldn't it? I'm not interested in working on it though.

**Comment by vurvdev** [comment]
Built in antispam?

I'd do this but would likely end up having to make a generic quota system like gohidas had an issue here for (which was closed).
